app/models/filters/filter_base.rb

```diff
@@ -652,8 +652,8 @@ def chosen(key)
         chosen_indefinite_disabilities
       when :dv_status
         chosen_dv_status
-      when :heads_of_household
-        yes_no(heads_of_household)
+      when :heads_of_household, :hoh_only
+        'Yes' if heads_of_household
```

Comment:
Should this be `'Yes' if heads_of_household || hoh_only`?

Reply:
Probably.  I believe the two are in sync, but I'll tweak that.  Thanks!